### PR TITLE
Update sample cfg to reflect options

### PR DESCRIPTION
**Pull request by rmarianski**
Since the tilequeue sample config was updated in https://github.com/tilezen/tilequeue/pull/113 I figured that I might as well do the same for tileserver.

@nvkelso, @iandees, @zerebubuth could you review please?

### CODE REVIEW

File: config.yaml.sample [hunk 1]
```diff
@@ -1,23 +1,50 @@
...
   user: osm
   password:
 queries:
   config: ../vector-datasource/queries.yaml
   template-path: ../vector-datasource/queries
+  # whether to reload the jinja query templates. This should be off in
+  # production.
   reload-templates: true
+
+# format extensions to support
```
Comment (nvkelso) [inline config.yaml.sample:18]: We should mention buffered MVT files are an option in the comment (but we don't support them in prod).

File: config.yaml.sample [hunk 2]
```diff
@@ -1,23 +1,50 @@
...
-# cache tiles locally
+
+# optional for health checks. This can be useful for monitoring to
+# verify that the service is still running and can connect to the
+# database.
+health:
+  # request path to listen for health checks
+  url: /_health
+
+# to cache tiles locally, enable a store. This can be useful when
```
Comment (nvkelso) [inline config.yaml.sample:41]: Nice! This answers a few support requests we've gotten this quarter.

File: config.yaml.sample (hunk 1 above)
Comment (nvkelso) [inline config.yaml.sample:18]: Fixed in 9bfdc30bfb8718bad8d0699da5e8ec22ce7ea93f.

**Comment by zerebubuth** [comment]
:+1:

**Pull request merged**